### DISCUSSION TITLE
Se mantendo atualizado em 2019

### DISCUSSION THREAD

**Discussion by devigor** [frontendbr/forum#1424] [Discussão]
Opa pessoal!
Quais blogs, fóruns, sites vocês costumam acompanhar para se manter atualizado com as novas tecnologias?? Utilizam de alguma ferramenta para ajudar nessa tarefa?

**Comment by Juuhz** [comment]
A princípio eu utilizo 3 canais:

1) Twitter
2) Este forum
3) https://tableless.com.br/

espero ter contribuído ;)

**Comment by gustavofsantos** [comment]
Também acompanho o dev.to, além disso uso esses sites:

- https://hacks.mozilla.org/
- https://web.dev/
- https://developers.googleblog.com/
- https://overreacted.io/ (mas o Dan não anda postando mais artigos)

Além disso também acompanho alguns subreddits

**Comment by felipefialho** [comment]
- [FrontendBR](https://github.com/frontendbr/forum) ❤️ 
- [Medium](https://medium.com/@felipefialho) (seguindo pessoas certas você recebe ótimos artigos) ✍️ 
- [Twitter](https://twitter.com/felipefialho_) (melhor rede, só seguir as pessoas certas) 🐦 
- Feedly para seguir [alguns conteúdos legais](https://github.com/felipefialho/frontend-feed)
- [MDN - Mozilla Developer Network](https://developer.mozilla.org)
- Lendo coisas aleatórias no [Stack Overflow](https://stackoverflow.com/)

**Comment by eustaquiofreitas** [comment]
Olá! Sou Eustáquio de Freitas Barbosa, tenho 39 anos, sou de Monte Azul - note de Minas Gerais... iniciante na linguagem javascript, decidido terminar minha vida escrevendo soluções.. claro assim que aprender o básico pra resolver problemas e ajudar pessoas...

**Comment by HmmerHead** [comment]
Poderiam dar algum exemplo, de algumas pessoas para se seguir no Twitter?

**Comment by andersoonweb** [comment]
@HmmerHead veja esse [link](https://uptodate.frontendrescue.org/pt/). Ele é um pouco "velho", mas tem boas pessoas.

**Comment by didiraja** [comment]
Além do fórum, minha **dica de ouro** com certeza são as [newsletter da CooperPress](https://cooperpress.com/publications/). Tem dos mais variados temas (Node, React, DB, Ruby), de acordo com o perfil do desenvolvedor.

Atualmente sigo _Front-End Focus_ e _Javascript Weekly_.

**Comment by allangrds** [comment]
Além dos links postados acima, acumulei alguns e fiz um post no Medium.

https://medium.com/trainingcenter/uma-lista-comovente-de-links-para-se-atualizar-e-estudar-sobre-tecnologia-4899e96dd78
